src/frontend/src/lib/components/hero/HeroContent.svelte

```diff
@@ -24,7 +24,10 @@
 </script>
 
 {#if summary}
-	<div transition:slide={{ delay: 0, duration: 250, easing: quintOut, axis: 'y' }}>
+	<div
+		transition:slide={{ delay: 0, duration: 250, easing: quintOut, axis: 'y' }}
+		class="flex flex-col items-center"
```

Comment:
Why not modying the existing flex element - the first child of this div?

Reply:
i tried at first but it didn't work; I even thought it was something with my current env. May I ask a favor? Can you try on your side and check if it works?

Reply:
I'll.

Reply:
as discussed I updated it